django/db/backends/base/schema.py

```diff
@@ -1617,6 +1617,8 @@ def _field_indexes_sql(self, model, field):
         return output
 
     def _field_should_be_altered(self, old_field, new_field, ignore=None):
+        if not old_field.column and not new_field.column:
+            return
```

Comment:
Can we check `.concrete` instead?

Reply:
OK

Reply:
https://github.com/django/django/pull/17240